modules/web/server/src/main/scala/navigate/web/server/http4s/WebServerLauncher.scala

```diff
@@ -84,7 +75,7 @@ object WebServerLauncher extends IOApp with LogInitialization {
           .optional
           .withFallback(ConfigSource.resources("app.conf").optional)
 
-  def makeContext[F[_]: Sync](tls: TLSConfig): F[SSLContext] = Sync[F].delay {
+  def makeContext[F[_]: Sync](tls: TLSConfig): F[SSLContext] = Sync[F].blocking {
```

Comment:
Nice. FS2 also has some [TLS](https://fs2.io/#/io?id=tls) stuff. Maybe in the future we could replace some code here with the non-blocking FS2 things. If you ever feel like a refactor 😄

Reply:
I just checked and odb uses blaze too. @tpolecat any info on why we are using blaze?